.github/workflows/stale.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  invalid-to-stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is invalid or waiting response because it has been open 7 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          days-before-issue-stale: 7
+          only-labels: 'invalid,awaiting response'
+
+  stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 5 days with no activity.'
```

Comment:
This issue is closed because it has been stale for 5 days. Please open a new issue if you have similar issues or you have any new updates now.